frontend/src/components/ScheduledEventsTableRow/ScheduledEventsTableRow.tsx

```diff
@@ -12,7 +17,46 @@ export interface ScheduledEventsTableRowProps {
   start_date: string;
 }
 
+interface RunInstancePayload {
+  start_at: Moment;
+  status: string;
+}
+
+const BASEPATH = api.routes.VIEW_SCHEDULED_EVENTS;
+
+const createRunInstance = async (
+  scheduleId: number | undefined,
+  data: RunInstancePayload,
+): Promise<boolean> => {
+  const token = await localForage.getItem<string>(localForageKeys.API_KEY);
+
+  return await axios.request({
+    url: `${BASEPATH}${scheduleId}${api.routes.CREATE_SCHEDULED_INSTANCE}`,
+    method: 'post',
+    headers: {
+      'Content-Type': 'application/json',
+      Authorization: `token ${token}`,
+    },
+    data,
+  });
+};
+
 export const ScheduledEventsTableRow: FunctionComponent<ScheduledEventsTableRowProps> = (props) => {
+  const [isCreatingInstance, setLoading] = useState(false);
+
+  const handleClick = () => {
+    setLoading(true);
+    createRunInstance(props.uid, {
+      start_at: moment(),
+      status: 'p',
+    })
+      .then(() => setLoading(false))
```

Comment:
You're assuming coming here always means a successful result ... The API will return a successful event even when authentication fails, so u need to handle those situations here by examining status codes.

Reply:
Now check for `200` and `201` status codes, otherwise we log the response